packages/react-devtools-shared/src/devtools/views/Components/InspectedElementContext.js

```diff
@@ -163,14 +182,11 @@ export function InspectedElementContextController({children}: Props) {
       inspectedElementRef.current !== inspectedElement
     ) {
       inspectedElementRef.current = inspectedElement;
-
-      if (typeof prefetchSourceFiles === 'function') {
-        prefetchSourceFiles(inspectedElement.hooks, fetchFileWithCaching);
-      }
```

Comment:
This optimization wasn't really important given the in-memory caching we now do, so I cleaned it up while I was in here.

Reply:
wouldn't this help speed things up in the case where the source files are loaded before devtools is open?